src/quazip.mk

```diff
@@ -0,0 +1,31 @@
+# This file is part of MXE.
+# See index.html for further information.
+PKG             := quazip
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.7.1
+$(PKG)_CHECKSUM := 78c984103555c51e6f7ef52e3a2128e2beb9896871b2cc4d4dbd4d64bff132de
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := http://$(SOURCEFORGE_MIRROR)/project/$(PKG)/$(PKG)/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc qtbase
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'http://sourceforge.net/projects/quazip/files/quazip/' | \
+    $(SED) -n 's,.*/\([0-9][^"]*\)/".*,\1,p' | \
+    head -1
+endef
+
+
+define $(PKG)_BUILD
+    cd '$(1)' && '$(PREFIX)/$(TARGET)/qt5/bin/qmake' PREFIX=$(PREFIX)/$(TARGET) $(if $(BUILD_STATIC), CONFIG\+=staticlib)
+    $(MAKE) -C '$(1)' -j '$(JOBS)'
+    $(MAKE) -C '$(1)' -j 1 install
+
+    '$(TARGET)-g++' \
+        -W -Wall -Werror -std=c++0x -pedantic \
+        $(if $(BUILD_STATIC), -D QUAZIP_STATIC) \
```

Comment:
Well, the quazip README [1] proposes to define this macro on the application side in case of static building (at least in my interpretation), and if it's not strictly necessary, i would avoid further patching of the quazip source tree. 

 What if we add this flag to the quazip.pc file for static targets, so at the application-side no need to explictly care about this?

[1]https://sourceforge.net/p/quazip/code/HEAD/tree/trunk/quazip/README.txt  line 41